.vitepress/theme/custom.css

```diff
@@ -0,0 +1,43 @@
+* {
+    -webkit-font-smoothing: antialiased;
+}
```

Comment:
We can add this to [theme](https://github.com/shopware/developer-documentation-vitepress/tree/main/src/shopware/styles).